content/en/docs/components/pipelines/sdk/connect-api.md

```diff
@@ -165,6 +166,9 @@ Choose your use-case from one of the options below:
                 audience: pipelines.kubeflow.org      
   ```
 
+  Note that this example uses `default-editor` in `my-namespace` as the service account identity, but you can configure
+  to use any service account that runs in your Pod.
```

Comment:
We are probably missing the explanation of what permissions the service account needs.

https://github.com/kubeflow/pipelines/issues/3513#issuecomment-721784849 is best documentation we have for now